vision/object_detection_segmentation/yolov2-coco/yolov2_coco_md5.txt

```diff
@@ -0,0 +1 @@
+MD5 (yolov2_coco.onnx) = dd60e6fb27a5956b7705c985e1f62737
```

Comment:
Additionally, MD5 file can be stored in the models folder (and the link on the README accordingly updated!) We're trying to standardize the storage of these files across all models.

Reply:
Thank you for your comment. I moved it.